esmvalcore/cmor/_fixes/cmip5/ec_earth.py

```diff
@@ -129,3 +130,59 @@ def fix_metadata(self, cubes):
         areacello.add_aux_coord(cube_to_aux_coord(lon), (0, 1))
 
         return iris.cube.CubeList([areacello, ])
+
+
+class Pr(Fix):
+    """Fixes for pr."""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata.
+        Fixes erroneously duplicated points in time coordinate 
+        Parameters
+        ----------
+        cube: iris.cube.CubeList
+        Returns
+        -------
+        iris.cube.CubeList
+        """
+        new_list = iris.cube.CubeList()
+        for cube in cubes:
+            try:
+                old_time = cube.coord('time')
+                if old_time.is_monotonic():
+                    pass
+
+                time_units = old_time.units
+                time_data = old_time.points
+
+                # erase none monotonic points (points erroneously copy-pasted)
+                d = np.diff(time_data)
+                idx_neg = np.where(d <= 0.)[0]
+                while len(idx_neg) > 0:
+                    time_data = np.delete(time_data, idx_neg[0] + 1)
+                    d = np.diff(time_data)
+                    idx_neg = np.where(d <= 0.)[0]
+
+                # create the new time coord
+                new_time = iris.coords.DimCoord(time_data, standard_name='time', var_name='time', units=time_units)
+
+                # create a new cube with the right shape
+                lat = cube.coord('latitude')
+                lon = cube.coord('longitude')
+                dims = (time_data.shape[0], lat.shape[0], lon.shape[0])
+                data = cube.data
+                new_data = np.ma.append(data[:dims[0]-1,:,:], data[-1,:,:])
+                new_data = new_data.reshape(dims)
+
+                tmp_cube = iris.cube.Cube(new_data, standard_name=cube.standard_name, long_name=cube.long_name,
+                                          var_name=cube.var_name, units=cube.units, attributes=cube.attributes,
+                                          cell_methods=cube.cell_methods,
+                                          dim_coords_and_dims=[(new_time, 0), (lat, 1), (lon, 2)])
+
+                new_list.append(tmp_cube)
+
+            except iris.exceptions.CoordinateNotFoundError:
+                pass
+
+        return new_list
```

Comment:
@remi-kazeroni a bit of a style not - it's best to have a try/except clause with as little code to try as possible - just one try condition gets executed anyway, and it also looks more readable: try: single clause / except (not clause): raise

Reply:
@valeriupredoi I have modified the fix to take your suggestion into account. Would that work for you now?

Reply:
looks good, cheers!